vendor/folio_deployment/README.md

```diff
@@ -0,0 +1,39 @@
+# FolioDeployment
```

Comment:
Should we nix this readme and keep everything in the project's readme, or write something meaningful here about the gem specfically?

Reply:
I'm thinking that this project will become the Gem eventually.